clientd3d/d3drender.c

```diff
@@ -7492,27 +7550,51 @@ void D3DRenderOverlaysDraw(d3d_render_pool_new *pPool, room_type *room, Draw3DPa
 							if (ROOM_OVERRIDE_DEPTH1 & GetRoomFlags())
 							{
 								depthf = GetOverrideRoomDepth(SF_DEPTH1);
+                        bUsingAlternateDepth = TRUE;

```

Comment:
Use "true" for booleans, not TRUE. Also, since this is set in every branch of the switch, it can be moved up to line 7546 and only set once.